calicoctl/commands/printer.go

```diff
@@ -230,7 +230,9 @@ func config(client client.Interface) func(string) string {
 		case "asnumber":
 			if asValue == "" {
 				if bgpConfig, err := client.BGPConfigurations().Get(context.Background(), "default", options.GetOptions{}); err != nil {
-					asValue = "unknown"
```

Comment:
Shouldn't we check if the error was due to the non-existence of this or an actual error accessing the data?  If the error was a failure to connect or something like that then I think the value should be unknown.

Reply:
Hrm, yeah I think you're right.

@jadech32 could you adjust to check the error type, and only show the default value if it's an `ErrorResourceDoesNotExist` (or whatever its called)?

For any other errors, we should continue showing "unknown"